README.md

```diff
@@ -32,20 +32,29 @@ A monorepo written in Go, represents [Keboola Connection](https://www.keboola.co
 
 ### Buffer API Benchmarking
 
-1. Start the server: `docker-compose run -p 8001:8001 --rm dev make run-buffer-api`
-1. Run the benchmark: `API_TOKEN=<token> docker-compose run k6 run /scripts/k6/<name>.js`
-
-Where `<token>` should be substituted with your Storage API token, and `<name>` with one of the following benchmark names:
-- `buffer-static` - Receiver with a single export. Export only has static columns.
-- `buffer-template` - Receiver with a single export. Export uses a template column.
+1. Start the server:
+```
+export KBC_STORAGE_API_HOST=connection.keboola.com
+docker-compose run -p 10000:8000 --rm dev make run-buffer-api-once
+```
+2. Run the benchmark:
+```
+export API_TOKEN=<token>
+export API_HOST=http://localhost:10000
+docker-compose run k6 run /scripts/k6/buffer-api/<name>
+```
```

Comment:
Honza pripravil nejake benchmarks pomocou `k6` nastroja.

Bolo to potrebne doladit, aby to spravne meralo priepustnost.
Samozne requesty boli ok, ale bol tam napr. zbytocny `sleep(1)` v benchmarku.

Vysledok.
![image](https://user-images.githubusercontent.com/19371734/210201180-8d0856fd-35d4-41c9-8c85-a94946ae7467.png)